vagrant/aws/aws-init.sh

```diff
@@ -18,16 +18,31 @@
 # or bring up your mini Kafka cluster.
 
 # Install dependencies
-sudo apt-get install -y maven openjdk-6-jdk build-essential \
-            ruby-dev zlib1g-dev realpath python-setuptools
+sudo apt-get install -y \
+  vagrant \
+  maven \
+  openjdk-8-jdk \
+  build-essential \
+  ruby-dev \
+  zlib1g-dev \
+  realpath \
+  python-setuptools
 
 base_dir=`dirname $0`/../..
 
-if [ -z `which vagrant` ]; then
-    echo "Installing vagrant..."
-    wget https://releases.hashicorp.com/vagrant/1.7.2/vagrant_1.7.2_x86_64.deb
-    sudo dpkg -i vagrant_1.7.2_x86_64.deb
-    rm -f vagrant_1.7.2_x86_64.deb
+# there is a bug with Vagrant 1.8 and Ruby 2.3
```

Comment:
Yeah, I wasn't suggesting sticking to 1.7.2, I was just saying you could use the existing manual install process and use a newer version of Vagrant (e.g. 1.9.3) if this issue has been fixed.